fake-delius-integration-api/__files/user-info-admin.json

```diff
@@ -0,0 +1,10 @@
+{
+  "username": "Art Major",
+  "email": "making-recall-decisions-tech@digital.justice.gov.uk",
```

Comment:
Probably better to not expose our email address to the internet.
Looks like you have copied this from elsewhere so it's already exposed, but I would suggest the emails are changed to example.com addresses.